copulas/univariate/base.py

```diff
@@ -81,11 +81,28 @@ def _select_candidates(cls, parametric=None, bounded=None):
 
         return candidates
 
+    @classmethod
+    def _get_subclasses(cls):
```

Comment:
I do not think this method is needed, since it is equivalent to calling the `_select_candidates` method above without any arguments.